src/sys/unix.rs

```diff
@@ -1391,6 +1391,39 @@ pub(crate) const fn to_mreqn(
     }
 }
 
+/// Possibles state for Path Maximum Transmission Unit (PMTU) Discovering of packets received on a socket.
+///
+/// It set in the IP packet Header the flag DF also known as "do not fragment".
+///
+/// For UDP it's particullary important to note that during Path MTU discovery is done by the OS,
+/// incomming datagram may be dropped.
+///
+/// ## Portability
+///
+/// It's Linux only way of managing the Path MTU.
```

Comment:
This is not needed as it's already shown in the docs, e.g. see https://docs.rs/socket2/latest/socket2/struct.Socket.html#method.pair.
```suggestion
```